kratos/processes/fast_transfer_between_model_parts_process.cpp

```diff
@@ -160,7 +161,7 @@ void FastTransferBetweenModelPartsProcess::TransferWithFlags()
 
         if (num_constraints != 0 && (mEntity == EntityTransfered::ALL || mEntity == EntityTransfered::CONSTRAINTS || mEntity == EntityTransfered::NODESANDCONSTRAINTS)) {
             const auto it_const_begin = mrOriginModelPart.MasterSlaveConstraintsBegin();
-            #pragma omp for
+            #pragma omp for schedule(guided, 512)
```

Comment:
Rather than adding new clauses, I'd modify existent ones to use the `ParallelUtils`.

Reply:
This is a fast PR for the aesthetic changes, in a different PR

Reply:
This is not aesthetic. I'll revert.

Reply:
OK, anyway is it possible to use critical with `ParallelUtils`

Reply:
Not yet.